applications/dashboard/models/class.assetmodel.php

```diff
@@ -79,7 +79,7 @@ public function addCssFile($filename, $folder = false, $options = false) {
      * @return array
      * @throws Exception
      */
-    public function getCssFiles($themeType, $basename, $eTag, &$notFound = null) {
+    public function getCssFiles($themeType, $basename, $eTag, &$notFound = null, $dynamicCurrentTheme = null) {
```

Comment:
Probably just `$currentTheme` is sufficient. Adding the word "dynamic" is just making it longer without adding information.

Reply:
Fine, will do.